packages/graph-retriever/src/graph_retriever/strategies/base.py

```diff
@@ -22,30 +44,34 @@ class Strategy(abc.ABC):
     Parameters
     ----------
     k :
-        Maximum number of nodes to retrieve during traversal.
+        Maximum number of nodes to select and return during traversal.
     start_k :
         Number of documents to fetch via similarity for starting the traversal.
         Added to any initial roots provided to the traversal.
     adjacent_k :
         Number of documents to fetch for each outgoing edge.
+    traverse_k :
+        Maximum number of nodes to traverse outgoing edges from before returning.
     max_depth :
         Maximum traversal depth. If `None`, there is no limit.
     """
 
     k: int = 5
```

Comment:
potentially change to `select_k`

Reply:
Didn't do this yet -- can do separately.